lib/ansible/modules/cloud/cloudstack/cs_vpc_offering.py

```diff
@@ -149,7 +149,9 @@ def get_vpc_offering(self):
         vo = self.query_api('listVPCOfferings', **args)
 
         if vo:
-            self.vpc_offering = vo['vpcoffering'][0]
+            for index, vpc_offer in enumerate(vo['vpcoffering']):
```

Comment:
any reason why we should not use a simplified loop?

~~~python
for vpc_offer in vo['vpcoffering']:
~~~

Reply:
Indeed, a simplified loop do the same.